CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+INCLUDE(CMakeForceCompiler)
+
+cmake_minimum_required (VERSION 2.8)
+SET(CMAKE_SYSTEM_NAME Generic)
+ 
+project(mbedTrace)
+
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include/)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/..//nanostack-libservice/mbed-client-libservice/)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/..//nanostack-libservice/)
+
+
+FILE(GLOB NANO_STACK_LIB_SERVICE_SRC
+	"${CMAKE_CURRENT_SOURCE_DIR}/source/*.c"
+
+	)
+
+CREATE_LIBRARY(mbedTrace "${NANO_STACK_LIB_SERVICE_SRC}" "") 
+
+
```

Comment:
Does this *really* have to be in the mbed-trace directory? Why not have it in the parent directory?

Right now it conflicts with the yotta build system that is still used for the library's unittests/CI checks.

```
16:02:32 -- Detecting CXX compiler ABI info
16:02:32 -- Detecting CXX compiler ABI info - done
16:02:32 
CMake Error at ../../CMakeLists.txt:20 (CREATE_LIBRARY):
16:02:32   Unknown CMake command "CREATE_LIBRARY".
16:02:32 
16:02:32 
16:02:32 -- Configuring incomplete, errors occurred!
16:02:32 See also "/builds/ws/mbed-trace-PR/15f02e58/build/x86-linux-native/CMakeFiles/CMakeOutput.log".
16:02:32 error: command ['cmake', '-D', 'CMAKE_BUILD_TYPE=RelWithDebInfo', '-G', 'Ninja', '.'] failed
16:02:33 Build step 'Execute shell' marked build as failure
16:02:33 [WARNINGS] Skipping publisher since build result is FAILURE
```